.connector-store/meta.json

```diff
@@ -14,17 +14,13 @@
         {
             "tagName": "v1.0.2",
             "products": [
-                "EI 6.6.0",
-                "EI 7.1.0",
                 "MI 4.1.0"
             ],
             "isHidden": false
         },
         {
             "tagName": "v2.0.0",
             "products": [
-                "EI 6.6.0",
-                "EI 7.1.0",
                 "MI 4.1.0"
```

Comment:
I think we have tested this connector with MI 4.3.0. Shall we check and update the product versions?

Reply:
Updated